minecraft_servers/warzone/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+    "server_name": "warzone",
+    "nice_name": "Warzone Network",
+    "direct_ip": "play.warzone.network",
+    "server_wildcards": [
+        "%.warzone.network",
+        "%.warz.one"
+    ],
+    "supported_languages": [
+        "en"
+    ],
+    "social": {
+        "web": "https://warzone.network",
```

Comment:
Many things just load indefinitely

Reply:
This is a known issue, the website is mostly used as a hub to our store, Discord invitation, and rules.